sunpy/timeseries/metadata.py

```diff
@@ -78,8 +86,10 @@ class TimeSeriesMetaData:
     def __init__(self, meta=None, timerange=None, colnames=None):
         self.metadata = []
         # Parse in arguments
-        if not isinstance(meta, type(None)):
-            if isinstance(meta, (dict, MetaDict)) and isinstance(timerange, TimeRange) and isinstance(colnames, list):
+        if meta is not None:
+            if (isinstance(meta, (dict, MetaDict)) and
+                isinstance(timerange, TimeRange) and
```

Comment:
It's weird to me that the indentation on this line and the one above is different.

Reply:
Otherwise PEP8 complains that the line below and the line below that are the same indentation level - not really sure what the fix is.